Scala/Scala.sublime-syntax

```diff
@@ -182,21 +182,31 @@ contexts:
           set: annotation-parameters
     - match: '(@)({{alphaplainid}})'
       captures:
-        1: meta.annotation.scala punctuation.definition.annotation.scala
-        2: meta.annotation.identifier.scala variable.annotation.scala
+       1: meta.annotation.scala punctuation.definition.annotation.scala
+       2: meta.annotation.identifier.scala variable.annotation.scala
       push: annotation-parameters
 
   annotation-parameters:
     - meta_content_scope: meta.annotation.parameters.scala
+    - match: (?=\(\))
```

Comment:
Is there a reason we need the lookahead here? Why not just `- match: \(\s*\)`?

Reply:
I did it for color consitency (without that, annotations like `@Inject()` have white braces)

Reply:
And `- match: \(\s*\)` doesn't allow for fixing that? I guess I'm just trying to clean up the nested context.